### DISCUSSION TITLE
add show_absolute in grid_power_169 and grid_ct_power

### DISCUSSION THREAD

**Discussion by R0DRIG0MEL0** [slipx06/sunsynk-power-flow-card#255] [General]
You could add the show_absolute option to grid_power_169 and grid_ct_power.

**Comment by slipx06** [comment]
I can do that for you

@R0DRIG0MEL0  Added to https://github.com/slipx06/sunsynk-power-flow-card/releases/tag/v4.16.0

**Comment by R0DRIG0MEL0** [comment]
Thank you very very much, it was perfect.
![Captura de tela 2024-02-18 161236](https://github.com/slipx06/sunsynk-power-flow-card/assets/60002358/a7cb5232-b5bb-4609-830e-1031428c5f47)
